src/commands/moderation/prefixCommands/functions/modifyCommand.ts

```diff
@@ -0,0 +1,205 @@
+    const aliasesString = interaction.options.getString('aliases')?.toLowerCase().trim() || '';
+    const aliases = aliasesString !== '' ? aliasesString.split(',') : [];
+    const isEmbed = interaction.options.getBoolean('is_embed');
+    const embedColor = interaction.options.getString('embed_color') || '';
+    const moderator = interaction.user;
+
+    const nameRegex = /^[\w-]+$/;
+    if (name && !nameRegex.test(name)) {
+        await interaction.followUp({ embeds: [wrongFormatEmbed(name)], ephemeral: true });
+        return;
+    }
+    for (const alias of aliases) {
+        if (!nameRegex.test(alias)) {
+            // eslint-disable-next-line no-await-in-loop
+            await interaction.followUp({ embeds: [wrongFormatEmbed(alias)], ephemeral: true });
+            return;
+        }
+    }
+    // Check if command name and alias are unique, additionally check if they do not exist as a version alias.
+    if (name) {
+        const foundCommandName = await PrefixCommand.findOne({
+            name: { $ne: command },
+            $or: [
+                { name },
+                { aliases: name },
+            ],
+        });
+        if (foundCommandName) {
+            await interaction.followUp({ embeds: [alreadyExistsEmbed(command, `${name} already exists as a different command or alias.`)], ephemeral: true });
+            return;
+        }
+        const foundVersion = await PrefixCommandVersion.findOne({
+            $or: [
+                { alias: name },
+            ],
+        });
+        if (foundVersion || name === 'generic') {
+            await interaction.followUp({ embeds: [alreadyExistsEmbed(command, `${name} already exists as a version alias.`)], ephemeral: true });
+            return;
+        }
+    }
+    if (aliases.length > 0) {
+        const foundCommandName = await PrefixCommand.findOne({
+            name: { $ne: command },
+            $or: [
+                { name: { $in: aliases } },
+                { aliases: { $in: aliases } },
+            ],
+        });
+        if (foundCommandName) {
+            await interaction.followUp({ embeds: [alreadyExistsEmbed(command, 'The new aliases contain an alias that already exists as a different command or alias.')], ephemeral: true });
+            return;
+        }
+        const foundVersion = await PrefixCommandVersion.findOne({
+            $or: [
+                { alias: { $in: aliases } },
+            ],
+        });
+        if (foundVersion || aliases.includes('generic')) {
+            await interaction.followUp({ embeds: [alreadyExistsEmbed(command, 'The new aliases contain an alias that already exists as a version alias.')], ephemeral: true });
+            return;
+        }
+    }
+
+    //Check if the mod logs channel exists
+    const modLogsChannel = interaction.guild.channels.resolve(constantsConfig.channels.MOD_LOGS) as TextChannel;
+    if (!modLogsChannel) {
```

Comment:
```suggestion
    let modLogsChannel = interaction.guild.channels.resolve(constantsConfig.channels.MOD_LOGS);
    if (!modLogsChannel || !modLogsChannel.isTextBased()) {
        modLogsChannel = null;
```
Might be safer to actually check if the channel is a text channel? Especially if we want to offer the codebase to other projects. Then, if it's not a text channel set it to null to prevent accidental attempts to send into a non-text channel.